common_define.h

```diff
@@ -125,6 +125,7 @@ typedef struct S_KUNBUS_REV_NUMBER {
 #define KUNBUS_FW_DESCR_TYP_PI_MIO                                118
 #define KUNBUS_FW_DESCR_TYP_PI_FLAT                               135
 #define KUNBUS_FW_DESCR_TYP_PI_CONNECT_4                          136
+#define KUNBUS_FW_DESCR_TYP_PI_PIRO                               137
```

Comment:
```suggestion
#define KUNBUS_FW_DESCR_TYP_PI_RIO                               137
```

Reply:
Does the thing also have inputs? It's more of a `RO`, isn't it?

Reply:
Damn you're right. Lack of coffee

Reply:
finally i got it.